src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -77,6 +77,21 @@ bool DisplayWindowMupen64plus::_start()
 {
 	FunctionWrapper::setThreadedMode(config.video.threadedVideo);
 
+#ifdef EGL
+	// Normally this is initialized externally through VidExt, but if VidExt is not implemented,
+	// do this here anyways. Calling eglInitialize has no negative effect according to the spec
+
+	EGLDisplay display;
+	if ((display = eglGetDisplay(EGL_DEFAULT_DISPLAY)) == EGL_NO_DISPLAY) {
+		LOG(LOG_ERROR, "eglGetDisplay() returned error %d", eglGetError());
+		return false;
+	}
+	if (!eglInitialize(display, nullptr, nullptr)) {
+		LOG(LOG_ERROR, "eglInitialize() returned error %d", eglGetError());
+		return false;
```

Comment:
This causes breakage on RPI4 and RPI3 in firmware kms mode, running in a KMS context via SDL2's kmsdrm driver. The plugin segfaults, leaving this message in the gliden64.log:
`2019/10/02,01:28:12.143,mupen64plus_DisplayWindow.cpp:90,ERROR, "eglInitialize() returned error 12289"`